tasks/copy_to_glacier/schemas/body.json

```diff
@@ -0,0 +1,145 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/copy_to_glacier/schemas/body.json",
+    "title": "copy_to_glacier SQS body schema",
+    "description": "Schema for metadata SQS body for copy_to_glacier lambda",
+    "type": "object",
+    "properties": {
+      "provider": {
+        "type": "object",
+        "description": "A dictionary containing provider that is in the ORCA holdings",
+        "properties": {
+          "id": {
+            "type": "string",
+            "description": "Provider ID supplied by Cumulus"
+          },
+          "name": {
+            "type": "string",
+            "description": "Name of the provider provided by Cumulus."
+          }
+        },
+        "required": [
+          "id",
+          "name"
+        ]
+      },
+      "collection": {
+        "type": "object",
+        "description": "A dictionary containing collection information",
+        "properties": {
+          "id": {
+            "type": "string",
+            "description": "Collection ID from Cumulus"
+          },
+          "shortname": {
+            "type": "string",
+            "description": "Collection short name from Cumulus."
+          },
+          "version": {
+            "type": "string",
+            "description": "Collection version from Cumulus."
+          }
+        },
+        "required": [
+          "id",
+          "shortname",
+          "version"
+        ]
+      },
+      "granule": {
+        "type": "object",
+        "description": "A dictionary containing granule information",
+        "properties": {
+          "granuleId": {
+            "type": "string",
+            "description": "Granule ID from Cumulus"
+          },
+          "executionId": {
+            "type": "string",
+            "description": "AWS step function execution id"
+          },
+          "cumulusCreateTime": {
```

Comment:
I concur

Reply:
Per Cumulus conversation, pattern is "^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])T(2[0-3]|[01][0-9]):([0-5][0-9]):([0-5][0-9])(.[0-9]*)?Z"

Reply:
I am not seeing where you have added the patterns for time. Per @AndrewDorn comment, since this is internal, you should be able to use our internal format unless Andrew wants something different for his database Lambda